### PR TITLE
Add versions for preCICE Distribution v2404.0

### PR DESCRIPTION
**Pull request by MakisH**
Still pending:

- [x] Release the VM
- [ ] Release this repository (self-referring, we can merge just before the release)

### CODE REVIEW

File: pages/docs/installation/installation-distribution.md [hunk 1]
```diff
@@ -14,6 +14,39 @@ publishing here lists of known-to-work versions.
...
+
+This is the first release that includes preCICE v3.x.
+
+It comprises the following components:
+
+- preCICE: [v3.1.1](https://github.com/precice/precice/releases/tag/v3.1.1)
+- Tools:
+  - ASTE: [v3.1.0](https://github.com/precice/aste/releases/tag/v3.1.0)
+  - config-visualizer: [v0.2.0](https://github.com/precice/config-visualizer/releases/tag/v0.2.0)
+  - config-visualizer-gui: commit [ae202e0](https://github.com/precice/config-visualizer-gui/tree/ae202e06ade780e02f31c8a88bb62de2c812ecdb) (new)
```
Comment (fsimonis) [inline pages/docs/installation/installation-distribution.md]: There are newer releases for both.

https://github.com/precice/config-visualizer/tags
https://github.com/precice/config-visualizer-gui/tags

The config-visualizer ~~needs another bugfix~~ was ~~fixed with v1.1.2~~ finally fixed with v1.1.3

Comment (fsimonis) [reply pages/docs/installation/installation-distribution.md]:
```suggestion
  - config-visualizer: [v1.1.3](https://github.com/precice/config-visualizer/releases/tag/v1.1.3)
  - config-visualizer-gui: [v0.1.0](https://github.com/precice/config-visualizer-gui/releases/tag/v0.1.0) (new)
```

Comment (MakisH) [reply pages/docs/installation/installation-distribution.md]: Why are these tags but not GitHub releases? This is confusing.

Comment (fsimonis) [reply pages/docs/installation/installation-distribution.md]: Done

File: pages/docs/installation/installation-distribution.md [hunk 2]
```diff
@@ -14,6 +14,39 @@ publishing here lists of known-to-work versions.
...
+This is the first release that includes preCICE v3.x.
+
+It comprises the following components:
+
+- preCICE: [v3.1.1](https://github.com/precice/precice/releases/tag/v3.1.1)
+- Tools:
+  - ASTE: [v3.1.0](https://github.com/precice/aste/releases/tag/v3.1.0)
+  - config-visualizer: [v1.1.3](https://github.com/precice/config-visualizer/releases/tag/v1.1.3)
+  - config-visualizer-gui: [v0.1.0](https://github.com/precice/config-visualizer-gui/releases/tag/v0.1.0) (new)
+  - ci-images: commit [b421a49](https://github.com/precice/ci-images/tree/b421a49a1ae94cbe02d3d1b79e38febddf5b7499)
```
Comment (uekerman) [inline pages/docs/installation/installation-distribution.md:30]:
```suggestion
  - ci-images: commit [b421a49](https://github.com/precice/ci-images/tree/b421a49a1ae94cbe02d3d1b79e38febddf5b7499)
  - FMI runner: [v0.2.1](https://github.com/precice/fmi-runner/releases/tag/v0.2.1)
  - Micro manager: [v0.4.0](https://github.com/precice/micro-manager/releases/tag/v0.4.0)
```

Comment (uekerman) [inline pages/docs/installation/installation-distribution.md]:
```suggestion
  - ci-images: commit [b421a49](https://github.com/precice/ci-images/tree/b421a49a1ae94cbe02d3d1b79e38febddf5b7499)
  - config-visualizer: [v1.1.3](https://github.com/precice/config-visualizer/releases/tag/v1.1.3)
  - config-visualizer-gui: [v0.1.0](https://github.com/precice/config-visualizer-gui/releases/tag/v0.1.0) (new)
  - FMI runner: [v0.2.1](https://github.com/precice/fmi-runner/releases/tag/v0.2.1)
  - Micro manager: [v0.4.0](https://github.com/precice/micro-manager/releases/tag/v0.4.0)
```

File: pages/docs/installation/installation-distribution.md [hunk 3]
```diff
@@ -14,6 +14,39 @@ publishing here lists of known-to-work versions.
...
+  - Matlab bindings: [v3.1.0](https://github.com/precice/matlab-bindings/releases/tag/v3.1.0) (now three-digit)
+  - Python bindings: [v3.1.0](https://github.com/precice/python-bindings/releases/tag/v3.1.0) (now three-digit)
+  - Rust bindings: [v3.1.0](https://github.com/precice/rust-bindings/releases/tag/v3.1.0) (new)
+- Adapters:
+  - CalculiX adapter: [v2.20.1](https://github.com/precice/calculix-adapter/releases/tag/v2.20.1)
+  - code_aster adapter: commit [b797fcc](https://github.com/precice/code_aster-adapter/tree/b797fccae996870ee9157b39e45f03d218f98452)
+  - deal.II adapter: [4c6d092](https://github.com/precice/dealii-adapter/tree/4c6d092c60c750478b08cfac25da1ff174c2d6f5)
+  - DUNE adapter: commit [75edcc3](https://github.com/precice/dune-adapter/tree/75edcc303510237505ed52eeab63bb746cbd3b44)
+  - DuMuX adapter: [v2.0.0](https://github.com/precice/dumux-adapter/releases/tag/v2.0.0) (new)
+  - FEniCS adapter: [v1.4.0](https://github.com/precice/fenics-adapter/releases/tag/v1.4.0)
```
Comment (uekerman) [inline pages/docs/installation/installation-distribution.md]:
```suggestion
  - FEniCS adapter: [v2.1.0](https://github.com/precice/fenics-adapter/releases/tag/v2.1.0)
```

File: pages/docs/installation/installation-distribution.md [hunk 4]
```diff
@@ -14,6 +14,39 @@ publishing here lists of known-to-work versions.
...
+  - OpenFOAM adapter: [v2.1.0](https://github.com/precice/openfoam-adapter/releases/tag/v2.1.0)
```
Comment (uekerman) [inline pages/docs/installation/installation-distribution.md]:
```suggestion
  - OpenFOAM adapter: [v1.3.0](https://github.com/precice/openfoam-adapter/releases/tag/v1.3.0)
```

Comment (MakisH) [reply pages/docs/installation/installation-distribution.md]: Good catch, thanks! Not sure what happened here...

**Comment by MakisH** [comment]
@uekerman ready to merge from my side. I can do that and then create a release tag (we are already on the main branch).

**Review by uekerman — APPROVED** [review]

**Pull request merged**